libs/@guardian/libs/src/consent-management-platform/onConsentChange.ts

```diff
@@ -67,7 +74,14 @@ const enhanceConsentState = (consentState: ConsentStateBasic): ConsentState => {
 		return {
 			...consentState,
 			canTarget: !consentState.ccpa.doNotSell,
-			framework: 'ccpa',
+			framework: 'usnat',
```

Comment:
Will this be the public framework set for 'ccpa' consent state?

We have a couple of instances of checking 'framework':
https://github.com/search?q=org%3Aguardian+%22%3D%3D%3D+%27ccpa%27%22&type=code

Reply:
CCPA has been completely replaced by usnat